base/setup/lib/utils/devutils.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * PROJECT:     ReactOS Setup Library
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Miscellaneous device utility functions
```

Comment:
```suggestion
 * PURPOSE:     Device utility functions
```
This word doen't contribute anything.